crates/sui-core/src/transaction_manager.rs

```diff
@@ -421,7 +419,9 @@ impl TransactionManager {
                     .expect("Failed to check if tx is already executed")
                 {
                     // also ensure the transaction will not be retried after restart.
-                    let _ = epoch_store.remove_pending_execution(&digest);
+                    epoch_store
+                        .remove_pending_execution(&digest)
+                        .expect("remove_pending_execution should not fail");
```

Comment:
`unwrap_or_else()` to print the error?

Reply:
Good point. Looking at this file though, it seems that we always just use .expect().
Might want to do a bigger refactoring to fix all of them